### PR TITLE
Bump Elixir, Erlang and NodeJS versions

### PR DESCRIPTION
**Pull request by Lucasnar**
It seems everything is working fine with these new versions. I tested some pages and they were good. Some more thorough regression tests would be nice to make sure, though.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ jobs:
   build:
     docker:
       # specify the version here
-      - image: circleci/elixir:1.8
+      - image: circleci/elixir:1.9
```
Comment (ricardoruwer) [inline .circleci/config.yml]: Change the Elixir and Erlang version in this file too: https://github.com/ricardoruwer/construtora_lc_hiert/blob/master/elixir_buildpack.config

Comment (Lucasnar) [reply .circleci/config.yml]: I think this should work: https://github.com/ricardoruwer/construtora_lc_hiert/pull/110/commits/21580039f491df002aa9f15538dff11dccd4ddfc. Is there any way we can test before this going to prod?

Comment (ricardoruwer) [reply .circleci/config.yml]: Yes, I'm deploying your code here https://qa-construtora-lc-hiert.herokuapp.com/ 👀

Comment (ricardoruwer) [reply .circleci/config.yml]:
```
-----> Fetching Erlang 22.0.7
-----> Installing Erlang 22.0.7 (changed)
-----> Fetching Elixir v1.9.1 for OTP 22
-----> Installing Elixir v1.9.1 (changed)
```
...

Comment (ricardoruwer) [reply .circleci/config.yml]:
```Will use the following versions:
       * Node 10.13.0```

Comment (ricardoruwer) [reply .circleci/config.yml]:
```
cp: cannot overwrite directory '/app/tmp/cache/node_modules/phoenix' with non-directory
cp: cannot overwrite directory '/app/tmp/cache/node_modules/phoenix_html' with non-directory
cp: cannot overwrite directory '/app/tmp/cache/node_modules/phoenix_live_view' with non-directory
 !     Push rejected, failed to compile Phoenix app.
 !     Push failed
```

Comment (ricardoruwer) [reply .circleci/config.yml]: I'll try this `clean_cache`: https://github.com/gjaldon/heroku-buildpack-phoenix-static/issues/80

Comment (ricardoruwer) [reply .circleci/config.yml]: Yeah, it worked now

Comment (ricardoruwer) [reply .circleci/config.yml]: @Lucasnar can you please add this line:
```clean_cache=true```
in the `phoenix_static_buildpack.config` file? 😄

Comment (Lucasnar) [reply .circleci/config.yml]: Sure! 

Glad it worked in the end!

**Pull request merged**